### ISSUE TITLE
Copy permissions bug

### ISSUE THREAD

**Issue by bricss** [gruntjs/grunt-contrib-copy#218]
Hi,

After updating to the latest version I'm getting this error in my builds with r.js:
```
Running "requirejs:compile" (requirejs) task
{ [Error: Error: EPERM, operation not permitted 'd:\Documents\Aptana Studio 3 Workspace\voot\assy\assets\js\main.min.js'
    at Error (native)
]
  originalError:
   { [Error: EPERM, operation not permitted 'd:\Documents\Aptana Studio 3 Workspace\voot\assy\assets\js\main.min.js']
     errno: -4048,
     code: 'EPERM',
     path: 'd:\\Documents\\Aptana Studio 3 Workspace\\voot\\assy\\assets\\js\\main.min.js',
     syscall: 'rename' } }
```
I've tried to set mode option to true, false and '0777' but nothing helps at all.
```javascript
copy : {
      main : {
        files : [{
          cwd : '<%= meta.rootPath %><%= meta.srcPath %>/',
          dest : '<%= meta.assyPath %><%= meta.srcPath %>/',
          dot : true,
          expand : true,
          src : ['**']
        }],
        options: {
          mode : true
        }
      }
    },
```

**Comment by bricss** [comment]
I found that the problem is coming from "file-sync-cmp", if comment it out then everything work properly.

**Comment by sindresorhus** [comment]
@mgeisler

**Comment by bricss** [comment]
Bug fixed: https://github.com/mgeisler/file-sync-cmp/pull/1

**Comment by mgeisler** [comment]
Thanks @bricss for the pull request! @sindresorhus I've released 0.1.1 with the fix.
